recipes-robot/fs-overlay/files/data.mount

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Mount Upper dir for overlay on dedicated partition
+
+[Mount]
+What=/dev/mmcblk0p4
+Where=/data
```

Comment:
Actually can you make `/var` the root mount point point? `/data` is a user-exposed directory.

Reply:
the overlay will be mounted on /var, but you need the upper mounted first, and it has to go somewhere, and it cannot be the same place, so maybe I can have the mount point be /mnt/var?